docs/compatibility.md

```diff
@@ -344,7 +344,6 @@ Due to such limitations, the input JSON schema must be `MAP<STRING,STRING>` and
  ```
 scala> val df = Seq("{}", "BAD", "{\"A\": 100}").toDF
 df: org.apache.spark.sql.DataFrame = [value: string]
-
```

Comment:
We really could benefit from a linter in the build. For this section I get the following violations:

- https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md032.md (which might be another reason why it's not displayed correctly )
- https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md007.md
- https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md031.md
- https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md040.md

Reply:
Hi @gerashegalov , thanks for the suggestion, I just create a [pr](https://github.com/NVIDIA/spark-rapids/pull/7929) to address and add the linter, there are more than 20 thousand errors in our doc if we enable all check rules, so I will just pick the rules which could raise formatting issue in our [webpage](https://nvidia.github.io/spark-rapids/),  and add it in the config file.

Reply:
even I only enable one MD032 rule, there still 140 errors in our document,  I checked some errors but none of them has formatting issue, not sure if we could benifit from this tool with not much effort.
![image](https://user-images.githubusercontent.com/84758614/227428982-e2a3ea62-8cb7-42c4-81f3-ac939c785a32.png)
https://github.com/NVIDIA/spark-rapids/actions/runs/4507932014/jobs/7936177393?pr=7929
@sameerz @gerashegalov any insight about this?

Reply:
since we have format issue on our [webpage](https://nvidia.github.io/spark-rapids/docs/compatibility.html#from_json-function) now, could we merge this firstly?

Reply:
yes, it makes sense to handle introduction of the linter in  a separate PR.